azure-spring-maven-plugin/src/main/java/com/microsoft/azure/maven/spring/AbstractSpringMojo.java

```diff
@@ -185,7 +185,7 @@ protected void initializeAuthConfiguration() throws MojoFailureException {
     }
 
     protected boolean isAuthConfigurationExist() {
-        final String pluginKey = String.format("%s:%s", plugin.getGroupId(), plugin.getArtifactId());
+        final String pluginKey = getPluginKey();
```

Comment:
plugin.getPluginLookupKey?

Reply:
Fixed

Reply:
plugin has the method of getPluginLookupKey